lib/chef/provider/aws_rds_instance.rb

```diff
@@ -36,14 +36,12 @@ def destroy_aws_object(instance)
     end
     # Wait up to 10 minutes for the db instance to shutdown
     converge_by "waited until RDS instance #{new_resource.name} was deleted" do
-      wait_for(
-        aws_object: instance,
-        query_method: :exists?,
-        expected_responses: [false],
-        acceptable_errors: [AWS::RDS::Errors::DBInstanceNotFound],
-        tries: 60,
-        sleep: 10
-      )
+      tries, sleep = 60, 10
```

Comment:
```wait_for``` does ```current_response = aws_object.send(query_method)``` so I don't think a proc will work. The code should basically wait for ```AWS::RDS::Errors::DBInstanceNotFound``` and I don't think ```wait_for``` can do that out of the box.